src/transaction_builder.js

```diff
@@ -110,6 +124,11 @@ export class TransactionBuilder {
 
     this.baseFee = opts.fee;
     this.timebounds = clone(opts.timebounds) || null;
+    this.ledgerbounds = clone(opts.ledgerbounds) || null;
+    this.minAccountSequence = opts.minAccountSequence || null;
+    this.minAccountSequenceAge = opts.minAccountSequenceAge || null;
+    this.minAccountSequenceLedgerGap = opts.minAccountSequenceLedgerGap || null;
+    this.extraSigners = opts.extraSigners || null;
```

Comment:
`clone()` equivalent? dunno

Reply:
Yeah, was debating if we need one for the array. Probably wouldn't hurt